client/libra-dev/build.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use bindgen::EnumVariation;
+use std::path::PathBuf;
+
+fn main() {
+    println!("cargo:rustc-link-lib=libra_dev");
+    println!("cargo:rustc-link-search=native=../libra-dev/target/debug/");
```

Comment:
This is wrong now, maybe we should  remove crate-type=["cdylib"]  from Cargo.toml , and remove line 8,9 here, since it is not needed for static library